docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,102 @@ keywords:
+Nexus has a streamlined Temporal developer experience, reliable execution, and integrated observability.
+
+- **Share a subset of a Temporal Application** \-
+Abstract and share a subset of an Application as a Nexus Service.
+Expose Services on a Nexus Endpoint for others to use and secure access with Endpoint access control policies.
+Nexus Endpoints decouple callers from handlers, so teams can operate more autonomously.
+
+- **Design for growth** \-
+Temporal Nexus enables a modular application design that can evolve as you grow.
+Start with Nexus Services in a monolithic Namespace and move Services to separate Namespaces with small configuration changes and no downtime for service consumers.
 
-For a deep dive into Temporal Nexus, see the [Temporal Nexus Encyclopedia section](https://docs.temporal.io/nexus).
-To connect with the Nexus community, join the [#nexus](https://temporalio.slack.com/archives/C07LQN0JK9B) channel in [Temporal Slack](https://t.mp/slack).
+- **Smaller failure domains** \- When teams operate in the same monolithic Namespace, everything is available to everyone, and mis-behaving workers can trigger rate limits that affect all teams operating in that monolithic Namespace.
+  Nexus enables each team to have their own Namespace for improved security, troubleshooting, and fault isolation.
 
-Temporal keeps the runtime benefits of an Event-Driven Architecture (EDA) where services can be down without affecting each other, but offers a higher-level durable execution programming model that lets you write idiomatic code in any language and preserve the state of execution.
-This ensures that your Workflows can be resumed automatically in the presence of failure, without the extensive boilerplate code and complexity that comes with using raw EDA frameworks directly.
+- **Multi-region** \-
+Nexus requests in Temporal Cloud are routed across a global mTLS-secured Envoy mesh.
+Built-in Nexus Machinery provides reliable at-least-once execution, even across multi-region boundaries.
 
-With Temporal Nexus, inter-service communication and durable execution can span teams, Namespaces, and regions all from within the Temporal context.
-Nexus preserves a familiar API programming model that supports both synchronous and asynchronous patterns, is suitable for low-latency and long-running operations, and uses an optimized form of the Temporal queue-based Worker architecture to preserve the benefits of EDA at runtime.
+### Advantages
 
-Temporal Nexus provides an integrated Temporal SDK experience, built-in Nexus Machinery, first-class observability, and enables each team to have their own Namespace for improved security, troubleshooting, and blast radius isolation.
-It allows multiple teams to collaborate autonomously while maintaining a clean separation of concerns, ensuring that applications are scalable, secure, and easy to manage as they grow.
+- Reliably connect Temporal Applications within and across Namespaces.
```

Comment:
I really like the update to the intro but this content looks like the same as the content currently on https://docs.temporal.io/cloud/nexus.

Reply:
That content is being removed in a follow-on PR #3196.